tdm_loader/tdm_loader.py

```diff
@@ -199,10 +200,11 @@ def channel_group_search(self, search_term):
 
         ind = []
         for name in found_terms:
-            i = chg_names.index(name)
-            ind.append((name, i))
+            for occurence in range(found_terms.count(name)):
+               i = self.channel_group_index(name, occurence)
+               ind.append((name, i))
```

Comment:
There is one space to less (probably from me when you just accepted my suggestion)

```suggestion
                i = self.channel_group_index(name, occurence)
                ind.append((name, i))
```